<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEvent</class>
 <widget class="QDialog" name="AddEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="LocationLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LocationInput">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="PhoneLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="PhoneInput">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="RPLabel3">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>RP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="RPInput">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="EventTypeLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Event Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EventTypeInput">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="CommentLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Initial Comment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="CommentInput">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SubmitButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
